docs/source/migrating_v1beta2_to_v1.rst

```diff
@@ -0,0 +1,138 @@
+from accepting an input project number to a project id:
+
+* datasets.list
+* jobs.search (now operations.list)
+* readgroupsets.export
+* variantsets.export
+
+This change impacts objects returned from APIs. The following objects
+have changed to contain a project id instead of a project number:
+
+* datasets
+* jobs (now operations)
+
+New datatypes
+=============
+
++---------------------+----------------------------+------------------------------+
+| Field               | v1beta2                    | v1                           |
++=====================+============================+==============================+
+| dataset.create_time | long                       | A timestamp in RFC3339 UTC   |
+|                     |                            | "Zulu" format                |
++---------------------+----------------------------+------------------------------+
+| read.info           | map<string, array<string>> | map<string, array<object>>   |
++---------------------+----------------------------+------------------------------+
+| annotations.info    | map<string, array<string>> | map<string, array<object>>   |
++---------------------+----------------------------+------------------------------+
+
+New objects
+===========
+
+Operations replaces jobs
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+The `jobs <https://cloud.google.com/genomics/v1beta2/reference/jobs>`_ object
+has been replaced with
+`operations <https://cloud.google.com/genomics/reference/rest/v1/operations>`_.
+
+Job Status
+~~~~~~~~~~
+
+With v1beta2, the status was encoded in the ``status`` field with
+detailed information in the detailedStatus.
+
+With v1, the status is a combination of ``status`` field plus the
+existence of the either the ``errors`` object (failure) *or* the
+``response`` object (success).
+
+ID
+~~
+
+With v1beta2, the job was identified with the ``id`` field.
+With v1, the operation is identified by the ``name`` field.
+
+importedIds
+~~~~~~~~~~~
+
+After a successful import, the job object would be populated with a list of
+``importedIds``.
+In the operation, this data is a child field of the ``response``, either the
+``callSetIds`` or ``readGroupSetIds``.
+
+created
+~~~~~~~
+
+The job ``created`` time is now in the operation ``metadata.createTime`` field.
+
+request
+~~~~~~~
+
+The job ``request`` values are now in the operation ``metadata.request`` field.
+
+
+Field Masks
+===========
+
+``get``, ``search``, and ``list`` APIs accept a list of fields to return,
+instead of all fields. This is sometimes referred to as a "field mask".
+Similarly ``patch`` APIs accept a field mask indicating the specific
+set of fields to change.
+
+In v1beta2, the "/" character could be used to separate a parent object
+from a child field, for example:
+
+* ``readGroupSets/id``
+
+In v1, the "/" character is not accepted, and parentheses must wrap the
+child field:
+
+* ``readGroupSets(id)``
+
+You can test building a proper field mask by using the "fields editor"
```

Comment:
That would be very useful, and they all have the same prefix: 

`https://developers.google.com/apis-explorer/#p/genomics/v1/genomics.`

And since they all have the `[category].[verb]` structure  (i.e. `readgroupsets.get`), it can be auto-scripted:

https://developers.google.com/apis-explorer/#p/genomics/v1/genomics.readgroupsets.get

Or the other way is to use directly the REST API, and [J]Query  through it:

https://www.googleapis.com/discovery/v1/apis/genomics/v1/rest

~p